gammapy/modeling/models/spectral.py

```diff
@@ -1572,137 +1596,20 @@ def read_builtin(cls, name, interp_kwargs=None):
         .. [3] Finke et al., "Modeling the Extragalactic Background Light from Stars and Dust"
             `Link <https://ui.adsabs.harvard.edu/abs/2010ApJ...712..238F>`__
 
-        Returns
-        -------
-        absorption : `Absorption`
-            Absorption model.
-
         """
         models = dict()
         models["franceschini"] = "$GAMMAPY_DATA/ebl/ebl_franceschini.fits.gz"
         models["dominguez"] = "$GAMMAPY_DATA/ebl/ebl_dominguez11.fits.gz"
         models["finke"] = "$GAMMAPY_DATA/ebl/frd_abs.fits.gz"
 
-        return cls.read(models[name], interp_kwargs=interp_kwargs)
-
-    def table_model(self, parameter):
```

Comment:
I guess one could convert it to a `PiecewiseNormSpectralModel`, but I'm not sure if there is any real use-case for this, so removing is fine I guess...

Reply:
It was used only in the test_absorbed_extrapolate, I just replace it using evaluate
https://github.com/gammapy/gammapy/blob/ba65d5aa7be4bd77f4778c48420db499dd8d28d8/gammapy/modeling/models/tests/test_spectral.py#L476